subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1722,13 +1747,70 @@ static inline int rx_demux_rx(memq_link_t *link, struct node_rx_hdr *rx)
 	return 0;
 }
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_BROADCASTER)
+static void term_evt_op_stop_cb(uint32_t status, void *param)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0, 0, &link, NULL, NULL};
+	struct ll_adv_set *adv;
+	struct ull_hdr *hdr;
+	uint32_t ret;
+
+	/* Ignore if race between thread and ULL */
+	if (status != TICKER_STATUS_SUCCESS) {
+		/* TODO: detect race */
+
+		return;
+	}
+	adv = param;
+	hdr = &adv->ull;
+	mfy.param = &adv->lll;
+
+	if (hdr->ref) {
+		LL_ASSERT(!hdr->disabled_cb);
+		hdr->disabled_param = NULL;
+		hdr->disabled_cb = NULL;
```

Comment:
As the assert above checks for being NULL, these assignment is not needed.

Reply:
It's done in ull_adv.c (ticker_op_stop_cb). Should we remove them there too? Or keep both?

Reply:
Actually, this is the function I removed.